### ISSUE TITLE
Bug: Doesn't support google memory store as Redis

### ISSUE THREAD

**Issue by sourabh-nutanix** [bosun-monitor/bosun#2478]
# Expected behaviour
It should support google memory store
> 

# Current behaviour

> Current bosun version doesn't support google memory store. Getting below error
with managed redis version 3.2
fatal: main.go:155: migrate.go:160: ERR unknown command 'CLIENT'
with version 4.0
fatal: main.go:155: migrate.go:160: ERR unknown command `CLIENT`, with args beginning with: `SETNAME`, `bosun`,
So basically CLIENT command is blocked by google memory store https://cloud.google.com/memorystore/docs/reference/redis-configs#blocked

## Context
 basically CLIENT command is blocked by google memory store https://cloud.google.com/memorystore/docs/reference/redis-configs#blocked

## Logs

> Please include any relevant log snippets or files here.

**Comment by sourabh-nutanix** [comment]
solved

**Comment by muffix** [comment]
Bosun supports Google Memory Store. 

For future reference: Google Memory Store doesn't allow any Redis `CLIENT` commands. Bosun can set a name for the Redis client connection which improves the readability of its output, but uses a `CLIENT` command to do so. 
Setting `RedisClientSetName = false` in the `DBConf` section of the Bosun config file prevents Bosun from doing that, so that it works with Google Memory Store.
